tools/pipelines/build-docs.yml

```diff
@@ -64,10 +64,10 @@ variables:
     value: true
   - name: isMain
     value: ${{ eq(variables['Build.SourceBranchName'], 'main') }}
-  - name: shouldDeploy
+  - name: forceDeployment
     value: ${{ or(
       eq(parameters.deployOverride, 'force'),
-      and(eq(variables.isMain, true), eq(parameters.deployOverride, 'default'))
+      eq(parameters.deployOverride, 'default'))
```

Comment:
It may be better to omit this variable altogether and just use `deployOverride` directly? `shouldDeploy` should automatically become true when this is `force`, and should automatically become false when this is `skip`.